helm/flannel-operator/templates/_resource.tpl

```diff
@@ -0,0 +1,24 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Create a name stem for resource names
+
+When pods for deployments are created they have an additional 16 character
+suffix appended, e.g. "-957c9d6ff-pkzgw". Given that Kubernetes allows 63
+characters for resource names, the stem is truncated to 47 characters to leave
+room for such suffix.
+*/}}
+{{- define "resource.default.name" -}}
+{{- "flannel-operator" -}}
```

Comment:
Here is a tricky part. There are lot of references to `flannel-operator` hard-coded in code. When we use release name, operator fails because of missing `flannel-operator` resources (e.g. some RBAC related stuff), because roles are now named as `flannel-operator-unique`. 
The code should be fixed with labels queries, but that's not my issue for now. I can add separate issue to update this later when Ludacris have enough time

Reply:
Fine with me for now. We hit similar issues on `kvm-operator`. There is some transition code needed because clusterroles can't have their name changed and so on.

This should not block you here and we will take care in the future.